packages/iov-bns/src/types.ts

```diff
@@ -487,3 +488,14 @@ export function isBnsTx(transaction: LightTransaction): transaction is BnsTx {
 
   return transaction.kind.startsWith("bns/");
 }
+
+export interface MultisigTransaction extends LightTransaction {
+  readonly multisig: readonly Uint8Array[];
+}
+
+export function isMultisigTransaction(transaction: LightTransaction): transaction is MultisigTransaction {
+  return (
+    Array.isArray((transaction as any).multisig) &&
```

Comment:
Did you test this for a non-MultisigTransaction input? I'd expect `transaction.multisig` to raise a reference error even before it is processed by `Array.isArray`. This can be avoided by a `typeof transaction.multisig === "object"`.

Reply:
hmm, looks like I am wrong when it comes to accessing object properties. I don't remember in which cases the ReferenceError occurs exactly. Feel free to ignore and resolve.